trillian/ctfe/configpb/config.proto

```diff
@@ -50,7 +50,13 @@ message LogConfig {
   // The public key is included for the convenience of test tools (and obviously
   // should match the private key above); it is not used by the CT personality.
   keyspb.PublicKey public_key = 5;
+  // If reject_expired is true then the certificate validity period will be
+  // checked during the validation of submissions. This will cause expired
```

Comment:
"checked against the current time during..." just to be clearer?

Reply:
Done.